Assets/Scripts/ToolHuman.cs

```diff
@@ -9,11 +9,15 @@ public class ToolHuman : Tool {
     // Start is called before the first frame update
     protected override void Start() {
         base.Start();
-        hapticFeedback = GetComponentInParent<HapticFeedback>();
+        if (!Application.isEditor) {
```

Comment:
I'm not sure but would it be better to check this `Application.isEditor` once and save it to a bool than call it every time?

Reply:
not sure either but a very valid suggestion

Reply:
for now since there's only 2 it should be fine. You can merge first.

Reply:
there's actually at least 6 so i changed it to also be consistent with handling oculus check. tested to work